app/timetables/models.py

```diff
@@ -1,3 +1,12 @@
 from __future__ import unicode_literals
 
 from django.db import models
+
+
+class Meal(models.Model):
+    name = models.TextField()
```

Comment:
If this field is not going to be implemented as a Textform input field, the let's use a CharField and specify a max_length while at it